compiler.go

```diff
@@ -371,6 +387,8 @@ func (t *task) release() {
 	}
 }
 
+var descriptorProtoPath = (*descriptorpb.FileDescriptorProto)(nil).ProtoReflect().Descriptor().ParentFile().Path()
```

Comment:
Any reason not to make this a constant `google/protobuf/descriptor.proto`?

Reply:
I had it that way originally. But then decided that maybe extracting this from the "source of truth" would be better than hard-coding a string in here. 🤷

Reply:
Changed to a constant and then added a test to make sure the constant matches this expression.